lib/tools/apk-utils.js

```diff
@@ -328,7 +385,7 @@ apkUtilsMethods.installOrUpgrade = async function (apk, pkg = null, timeout = 60
 
   if (!await this.isAppInstalled(pkg)) {
     log.debug(`App '${apk}' not installed. Installing`);
-    await this.install(apk, false, timeout);
+    await this.install(apk, Object.assign({replace: false}, options));
```

Comment:
I think this should be `Object.assign({}, options, {replace: false})` otherwise someone could pass in `replace: true` and it would overwrite what you have.